doc/unison-manual.tex

```diff
@@ -132,31 +132,37 @@
 \URL{http://research.microsoft.com/\home{fournet}/}{Cedric Fournet}
 contributed to an even earlier prototype.
 
-\TOPSUBSECTION{Mailing Lists and Bug Reporting}{lists}
+\TOPSUBSECTION{Obtaining Unison}{obtaining}
 
-\input{contactsbody}
+Unison is primarily distributed as source code, which contains
+instructions in {\tt INSTALL.md}:
+\begin{quote}
+\ONEURL{https://github.com/bcpierce00/unison}
+\end{quote}
+
+The Unison wiki contains information about builds done as part of
+Continuous Integration and other sources of binaries; read the entire
+wiki at:
+\begin{quote}
+\ONEURL{https://github.com/bcpierce00/unison/wiki}
+\end{quote}
+
+\TOPSUBSECTION{Community, Maintenance, and Development}{development}
```

Comment:
See how this section looks when rendered. Using `\paragraph`, like in the old contactsbody.tex, looks slightly better in my opinion.

Reply:
adjusted